app/frontend/styles/_button.scss

```diff
@@ -1,4 +1,5 @@
-// TODO: Remove once implemented cookie banner from design system
-.app-button--cookies {
-  min-width: 12em;
+.app-button-group--align-center {
+  .govuk-button-group {
+    align-items: center;
+  }
```

Comment:
```suggestion
// TODO: Remove once cookie banner implemented using JS (i.e. without `button_to` form buttons)
.govuk-cookie-banner .govuk-button-group {
  align-items: center;
```

Reply:
…and we should rename this file `_cookie-banner.scss`

Reply:
👍